docs/source/form/select/components/organization-select.md

```diff
@@ -0,0 +1,47 @@
+---
+title: <AvOrganizationSelect />
+summary: A select list that automatically loads and pages through organizations when the user scrolls down.
+---
+
+## Example
+
+```jsx live=true viewCode=true
+import { Form } from '@availity/form';
+import { AvOrganizationSelect } from '@availity/select/resources';
+import * as yup from 'yup';
+import '@availity/yup';
+
+<Form
+  initialValues={{
+    organizations: '',
+  }}
+  onSubmit={values => apiResource.submit(values)}
+  validationSchema={yup.object().shape({
+    organizations: yup.string().isRequired('This field is required.'),
+  })}
+>
+  <AvOrganizationSelect
+    id="organization"
+    name="organization"
+    parameters={{
+      permissionId:['1111', '2222'],
+      regionId: 'FL'
+    }}
+    additionalPostGetArgs={{
+      resourceIds: ['1234', '4321']
+    }}
+  />
+
+  <Button color="primary" type="submit">
+    Submit
+  </Button>
+</Form>
+```
+
+## Props
+
+Extends [ResourceSelect Props](/form/select/components/resource-select/#props).
+
+### `additionalPostGetArgs?: object`
```

Comment:
I think this is a bit confusing. `additionalPostGetArgs` is a prop from `ResourceSelect`. We've already stated we're extending the `ResourceSelect` props.

We definitely need to call out how to filter by resource id, however. I think we either need to:
1. Change how this documentation is represented. Rather than stating that `AvOrgananizationSelect` has its own prop, just make a note to include `resourceIds` in the `additionalPostGetArgs`
2. Make a separate `AvOrganizationSelect` with a custom `resourceIds` prop that is a convenience helper for it being passed into `additionalPostGetArgs`.